go-controller/pkg/ovn/ovn.go

```diff
@@ -276,8 +279,55 @@ func (oc *Controller) Run(wg *sync.WaitGroup) error {
 	oc.WatchNodes()
 
 	oc.WatchPods()
-	oc.WatchServices()
-	oc.WatchEndpoints()
+
+	// Services are handled differently depending on the Kubernetes API versions
+	// We use a level triggered if k8s > 1.20, using endpoint slices instead endpoints
+	if util.DetectEndpointSlices(oc.client) {
+		// Services are handled differently depending on the Kubernetes API versions
+		klog.Infof("Using EndpointSlices instead of Endpoints in k8s versions > 1.19")
```

Comment:
comment and log message claim different versions

Reply:
which format is better to understand >= 1.20, >1.19, 1.20+ ???